generate_workspace/BUILD

```diff
@@ -1,9 +0,0 @@
-load('//:generate_workspace.bzl', 'generated_java_libraries')
```

Comment:
Removed the file, per your recommendations. However, with this removed, where is the java_binary defined?